code/modules/projectiles/updated_projectiles/projectile.dm

```diff
@@ -394,17 +401,19 @@
 	. = max(5, .) //default hit chance is at least 5%.
 	if(lying && stat) . += 15 //Bonus hit against unconscious people.
 
-	if(isliving(P.firer))
-		var/mob/living/shooter_living = P.firer
+	if(istype(P.firer, /mob/living/carbon))
```

Comment:
```suggestion
	if(iscarbon(P.firer))
```

Reply:
Or just type cast it with `var/mob/living/carbon/shooter_living = P.firer` and then do a `if(istype(shooter_living))`